tests/all/debug/translate.rs

```diff
@@ -24,6 +24,25 @@ fn check_wasm(wasm_path: &str, directives: &str) -> Result<()> {
     Ok(())
 }
 
+#[allow(dead_code)]
```

Comment:
This is a copy&pasted function which could be refactored. I didn't do it, because this is test-only code. If somebody objects, I'm happy to fix.